docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/media-tracking/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Media tracking"
+sidebar_position: 100
+---
+
+# Media tracking
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Media from "@site/docs/reusable/media/_index.md"
+```
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="ios" label="iOS" default>
+
+<Media tracker="ios" />
+
+### Auto-tracking events from AVPlayer
```

Comment:
this section is only visible for Swift, is that intentional?

Reply:
Yes, because the auto-tracking is only provided on iOS with the `AVPlayer` video player.